aws/codegen-aws-sdk/src/main/kotlin/software/amazon/smithy/rustsdk/AccountIdEndpointParamsDecorator.kt

```diff
@@ -234,6 +238,38 @@ class AccountIdEndpointModeBuiltInParamDecorator : ConditionalDecorator(
                                 )
                             }
                         }
+
+                        override fun trackSdkFeatures(
+                            codegenContext: ClientCodegenContext,
+                            configBag: String,
+                        ) = writable {
+                            rustTemplate(
+                                """
+                                match cfg
```

Comment:
This could probably be turned into a `pub(crate)` function that is re-used which would reduce code size a bit since it applies to every operation.

Also for some reason this code block is [rendered](https://d2luzm2xt3nokh.cloudfront.net/codegen-diff/1fe10969f786b1b53e4e1b8a40a2994e369566a4/27102829f69b2cfe3a9ca9e69ae64d1ea40f9865/aws-sdk/index_page_15.html#file-15-4) in between endpoint params.